lib/omnibus-ctl.rb

```diff
@@ -694,6 +694,11 @@ def run(args)
 
       command_to_run = args[0]
 
+      if command_to_run == "--help"
+        help
+        Kernel.exit 0
+      end
```

Comment:
Also, just curious-- I understand tests break if we add `--help` as a command, but I'm not sure why/how